catalog/nsili/catalog-nsili-endpoint/src/test/java/org/codice/alliance/nsili/endpoint/TestOrderMgrImpl.java

```diff
@@ -0,0 +1,270 @@
+        protoValueAny.insert_string("https");
+        protocolProp.value = protoValueAny;
+
+        NameValue portProp = new NameValue();
+        portProp.aname = "PORT";
+        Any portValueAny = orb.create_any();
+        portValueAny.insert_long(8993);
+        portProp.value = portValueAny;
+
+        NameValue[] properties = new NameValue[] {protocolProp, portProp};
+        orderMgr.order(null, properties);
+    }
+
+    @Test
+    public void testGetTimeout() throws Exception{
+        int timeout = orderMgr.get_timeout(null);
+        assertThat(timeout, is(AccessManagerImpl.DEFAULT_TIMEOUT));
+    }
+
+    @Test
+    public void testGetDefaultTimeout() throws Exception {
+        orderMgr.set_default_timeout(5000);
+        int timeout = orderMgr.get_default_timeout();
+        assertThat(timeout, greaterThan(0));
+    }
+
+    @Test
+    public void testGetUseModes() throws Exception {
+        String[] useModes = orderMgr.get_use_modes();
+        assertThat(useModes, notNullValue());
+        assertThat(useModes, arrayContainingInAnyOrder("OrderAccess"));
+    }
+
+    @Test
+    public void testQueryAvailDelay() throws Exception {
+        int delay = orderMgr.query_availability_delay(null, null, null);
+        assertThat(delay, greaterThan(0));
+    }
+
+    @Test
+    public void testNumberOfPriorities() throws Exception {
+        int numPriorities = orderMgr.get_number_of_priorities();
+        assertThat(numPriorities, is(1));
+    }
+
+    @Test
+    public void testGetActiveRequests() throws Exception {
+        Request[] activeReqs = orderMgr.get_active_requests();
+        assertThat(activeReqs.length, is(0));
+    }
+
+    @Test
+    public void testSetAvailability() throws Exception {
+        SetAvailabilityRequest request = orderMgr.set_availability(null, null, null, (short)1);
+        assertThat(request, notNullValue());
+    }
+
+    @Test
+    public void testGetPropertyNames() throws Exception {
+        String[] properties = orderMgr.get_property_names();
+        assertThat(properties.length, is(2));
+    }
+
+    @Test (expected = NO_IMPLEMENT.class)
+    public void testGetPropertyValues() throws Exception {
+        orderMgr.get_property_values(null);
+    }
+
+    @Test (expected = NO_IMPLEMENT.class)
+    public void testGetLibraries() throws Exception {
+        orderMgr.get_libraries();
+    }
+
+    private void setupOrderMgrMocks() throws Exception {
+        int testTotalHits = 5;
+        List<Result> results = new ArrayList<>(testTotalHits);
+        MetacardImpl testMetacard = new MetacardImpl();
+        testMetacard.setId(testMetacardId);
+        Result testResult = new ResultImpl(testMetacard);
+        results.add(testResult);
+        QueryResponse testResponse = new QueryResponseImpl(null, results, testTotalHits);
+        when(mockCatalogFramework.query(any(QueryRequest.class))).thenReturn(testResponse);
+    }
+
+    private void setupOrb()
```

Comment:
Good catch. I refactored that into the TestNsiliCommon that I setup for some of those test common items like that.